tasks/release.yaml

```diff
@@ -0,0 +1,15 @@
+---
+platform: linux
+image_resource:
+  type: docker-image
+  source:
+    repository: golang
+    tag: '1'
+inputs:
+  - name: src-tags
```

Comment:
It might be better to call this `src` still, and just do an input mapping in the plan?

Reply:
Not clear what you are asking for here.

The `src` resource is the unfiltered commit stream from `master`.
The `src-tags` resource is the same, filtered to tagged commits.

Ah. I am thinking of a different level.

You are in the task definition, where the input is mapped to the path.
You want the input renamed, and the in the pipeline definition another mapping from `src-tags` to `src`. ... Found the docs http://concourse-ci.org/task-step.html#input_mapping

Ok, will do.